komodo-relational/src/main/java/org/komodo/relational/teiid/CachedTeiid.java

```diff
@@ -72,10 +89,10 @@
          */
         @Override
         public CachedTeiid create( final UnitOfWork transaction,
-                             final Repository repository,
-                             final KomodoObject parent,
-                             final String id,
-                             final RelationalProperties properties ) throws KException {
+                                   final Repository repository,
+                                   final KomodoObject parent,
+                                   final String id,
+                                   final RelationalProperties properties ) throws KException {
             return new CachedTeiidImpl(transaction, repository, id);
```

Comment:
guys, deliberate re-formatting is fine and I think encouraged.. and I agree on turning off reformat-on-save ... that's just a bad feature.  

We can debate the "standard/project" formatting rules and develop another sharable set of Eclipse Java preferences for Komodo.. if we haven't already.

So unless someone mistakenly commits a "bunch" of reformatting.. I'd rather spend our time on checking our code logic and architecture.  We still have a lot of work to do in a short period of time.